Podfile

```diff
@@ -198,7 +198,7 @@ target 'WordPress' do
 
     pod 'Gridicons', '~> 1.0.2'
 
-    pod 'WordPressAuthenticator', '~> 1.29.0'
+    pod 'WordPressAuthenticator', '~> 1.30.0-beta.2'
```

Comment:
We don't include the beta number, so it should just be `1.30.0-beta`.

Reply:
Oh, I missed that, thanks!